core/src/main/java/org/dotwebstack/framework/core/scalars/DateCoercing.java

```diff
@@ -34,6 +34,11 @@ public LocalDate serialize(@NonNull Object value) {
     }
 
     try {
+      // to be able to also convert dates that also contain times
+      if (((String) value).contains("T")) {
+        value = ((String)value).substring(0, ((String)value).lastIndexOf('T'));
```

Comment:
shadowing is slecht
```suggestion
        String datePart = ((String)value).substring(0, ((String)value).lastIndexOf('T'));
        return LocalDate.parse(datePart);
```
Anders eerst als localdatetime en dan omzetten?
```suggestion
        LocalDateTime localDateTime = LocalDateTime.parse(value);
         return localDateTime.toLocalDate();
```